### DISCUSSION TITLE
extract SWIFT code

### DISCUSSION THREAD

**Discussion by red-sight** [Simplify/ibantools#61] [Ideas]
Is it possible to extract the SWIFT code from the IBAN or BBAN?

**Comment by Simplify** [comment]
Not really. Only up to first 6 characters.

SWIFT/BIC: [2 letter country code][4 letter swift code][2 letter location code][optional 3 letter branch code]
IBAN starts with 2 letter country code. First 4 letters in BBAN are usually 4 letter swift code. There is no way to find 2 letter location code that is required.

**Reply by red-sight** [reply]
Thank you for the answer
